rich-default/environment.yml

```diff
@@ -0,0 +1,522 @@
+name: base
+channels:
+  - samoturk
+  - mordred-descriptor
+  - rdkit
+  - conda-forge
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _r-mutex=1.0.1=anacondar_1
```

Comment:
I'd remove these two that start with underscores.

Reply:
The build works now with those dependencies added in.